### PR TITLE
LPS-122757 Add model resource

### PR DESCRIPTION
**Pull request by carolmariaabb**
Related Issue: https://issues.liferay.com/browse/LPS-122757

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp [hunk 1]
```diff
@@ -124,6 +126,12 @@ renderResponse.setTitle((formInstance == null) ? LanguageUtil.get(request, "new-
 		<aui:input name="serializedFormBuilderContext" type="hidden" value="<%= serializedFormBuilderContext %>" />
 		<aui:input name="serializedSettingsContext" type="hidden" value="" />
 
+		<aui:input name="dataDefinition" type="hidden" />
+		<aui:input name="dataLayout" type="hidden" />
+		<aui:input name="dataDefinitionId" type="hidden" value="<%= ddmStructureId %>" />
+
+		<aui:model-context bean="<%= ddmStructure %>" model="<%= DDMStructure.class %>" />
```
Comment (carolmariaabb) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/edit_form_instance.jsp:133]: Não sei se é necessário..

**Comment by liferay-forms** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by carolmariaabb** [comment]
ci:stop

**Comment by liferay-forms** [comment]
Hey @carolmariaabb, can you please check if these failures are related to your changes? Thanks

**Pull request closed without merging**